docs/vision-usage.rst

```diff
@@ -0,0 +1,233 @@
+Using the Vision API
+====================
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
```

Comment:
This seems like common stuff. Is there no way to do `include` of a common shared file?